scripts/build

```diff
@@ -4,4 +4,5 @@ source $(dirname $0)/lib/debug_functions
 source $(dirname $0)/lib/version
 
 $(dirname $0)/build-operator
+$(dirname $0)/generate-embeddedyamls
```

Comment:
I left the generate-embedded yamls out of the build-subctl to make sure it's not re-generated always. Although, I guess it's very quick, and probably could make sense to get rid of the makefile logic around that, and simply throw the generate inside build-subctl